include/amici/model.h

```diff
@@ -79,6 +79,8 @@ class Model : public AbstractModel {
      * @param nx_solver number of state variables with conservation laws applied
      * @param nxtrue_solver number of state variables of the non-augmented model
      * with conservation laws applied
+     * @param nx_solver_reinit number of state variables with conservation laws
+     * applied which would be reinitiliazed after preequilibration
```

Comment:
```suggestion
     * subject to reinitialization
```
(also includes presimulation)